assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -3305,6 +3307,7 @@ protected String build_instructor_grade_submission_context(VelocityPortlet portl
             final String submitterNames = users.values().stream().map(u -> u.getDisplayName() + " (" + u.getDisplayId() + ")").collect(Collectors.joining(", "));
             context.put("submitterNames", formattedText.escapeHtml(submitterNames));
             context.put("submissionStatus", assignmentService.getSubmissionStatus(s.getId()));
+            context.put("submitterId", s.getSubmitters().iterator().next().getSubmitter());
```

Comment:
This is probably safer:
`s.getSubmitters().stream().findAny().ifPresent(u -> context.put("submitterId", u.getSubmitter()));`

Reply:
@ern Done!